packages/map/src/hooks/useResizeObserver.ts

```diff
@@ -12,15 +12,14 @@ export const useResizeObserver = (isEditor: boolean) => {
   const resizeObserver = new ResizeObserver(entries => {
     for (let entry of entries) {
       let { width, height } = entry.contentRect
-      let newViewport = getViewport(entry.contentRect.width)
 
-      let editorWidth = 350
+      const EDITOR_WIDTH = 350
```

Comment:
I think this might be a good example of where renaming the fontSettings.ts file you've created to something more generic like constants.ts would be helpful at the core level.